### DISCUSSION TITLE
邦有道，危言危行；邦无道，危行言孙。

### DISCUSSION THREAD

**Discussion by BigRoseAndBoss** [WeNeedHome/SummaryOfLoanSuspension#519] [General]
RT

**Comment by zxcs001** [comment]
别搞这么多虚头巴脑的东西，知道为什么抖音、西瓜视频、头条上粉红为什么那么多么？就是因为很多自认为自己有文化的人，不爱说“人”话。

直接说大白话，很简单。

你共产党，丧尽天良，不守诚信，把老百姓的血汗钱都霍霍了。生育率搞的世界倒数，对汉人自己人种族灭绝，罪该万死。到时候，倒台的时候，别以为钱拿出去就能跑的了。不把老百姓的民脂民膏还回来，你即使把钱放瑞士银行里面，你人和子孙后代也会被特工给绑回来，碎尸万段。

**Comment by BigRoseAndBoss** [comment]
顶楼
